app/views/MainView/index.tsx

```diff
@@ -11,9 +11,9 @@ import { CircularProgress, Snackbar } from '@material-ui/core';
 import { $httpProvider } from 'ngimport';
 
 import AuthenticatedRoute from '../../components/AuthenticatedRoute';
-import SecurityContext from '../../components/SecurityContext';
-import EditContext from '../../components/EditContext';
-import SnackbarContext from '../../components/SnackbarContext';
+import SecurityContext from '../../context/SecurityContext';
+import EditContext from '../../context/EditContext';
+import SnackbarContext from '../../context/SnackbarContext';
```

Comment:
non-blocker, but 
@ imports ...? you went through the effort of changing and moving them